Pod/Utilities.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Utilities.swift
+//  ElJayKit
+//
+//  Created by Isaac Greenspan on 12/10/15.
+//  Copyright © 2015 Isaac Greenspan. All rights reserved.
+//
+
+import Alamofire
+
+/// Facilitate generics that Swift doesn't directly allow.
+public struct Of<T> {
```

Comment:
Can you explain more of the reasoning behind this?  Is it so that you don't need to create type aliases for specific callback function signatures?
eg: `BaseCallback` is equivalent to this: `public typealias anyCallback = (Result<Any, API.Error>) -> Void`, right?
To me, it's confusing to see `Of<some type>.callback` and have to hunt down `Of` to fugue out what's going on.  I'd prefer this to be renamed, or to have specific type aliases for each result type to be more explicit.